server/commiter.go

```diff
@@ -84,22 +89,27 @@ func (p *Plugin) executeCommiterCommand(commandArgs []string, args *model.Comman
 		return appErr
 	}
 
-	go p.updateCommitersPost(loadingPost, args.UserId, owner, repo, since, until)
+	go p.updateCommitersPost(loadingPost, args.UserId, owner, repo, isOrg, since, until)
 
 	return nil
 }
 
-func (p *Plugin) updateCommitersPost(post *model.Post, userID, org, repo string, since, until time.Time) {
+func (p *Plugin) updateCommitersPost(post *model.Post, userID, org, repo string, isOrg bool, since, until time.Time) {
 	// Fetch commits until one day after at midnight
 	fetchUntil := until.AddDate(0, 0, 1).Add(-time.Microsecond)
 
 	var commits []*github.RepositoryCommit
 	var err error
 	if repo != "" {
 		commits, err = p.fetchCommitsFromRepo(org, repo, since, fetchUntil)
-	} else {
-		commits, err = p.fetchCommitsFromOrg(org, since, fetchUntil)
 	}
+	switch isOrg {
+	case false:
+		commits, err = p.fetchCommitsFromUser(org, since, fetchUntil)
+	default:
+		p.fetchCommitsFromOrg(org, since, fetchUntil)
+	}
+
```

Comment:
Sorry for be ambiguous here. I had something like a switch for all three cases in mind:

```golang
	switch {
	case repo != "":
		commits, err = p.fetchCommitsFromRepo(org, repo, since, fetchUntil)
	case isOrg:
		commits, err = p.fetchCommitsFromOrg(org, since, fetchUntil)
	case !isOrg:
		commits, err = p.fetchCommitsFromUser(org, since, fetchUntil)
	}
```